specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-08-03-preview/managedClusters.json

```diff
@@ -3522,10 +3522,46 @@
         "windowsProfile": {
           "$ref": "#/definitions/AgentPoolWindowsProfile",
           "description": "The Windows agent pool's specific profile."
+        },
+        "networkProfile": {
```

Comment:
Not sure if this would lead to some misunderstanding, since there is already a property named `networkProfile` in mc.

Reply:
This one is in agent pool so I think it's good to call it networkProfile. AgentPoolNetworkProfile sounds redundant.